### PR TITLE
Hide sections and display warning if no features data is available

### PR DESCRIPTION
**Pull request by abbiefarr**
## Issue

Addresses #330 

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Verifies that the listing has any "Features" data. If not, displays a warning that no data is available. E.g:

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

I manually deleted data from the seeded DB so there were no units or feature information and verified that the warning was displayed and no sections were visible (see photo above).

- [x] Desktop View
- [ ] Mobile View

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes

### CODE REVIEW

File: sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationAddress.tsx [hunk 1]
```diff
@@ -14,6 +14,8 @@ const DetailApplicationAddress = () => {
       return t("application.contact.mailingAddress")
   }
 
+  if (!listing.leasingAgentAddress) return <></>
```
Comment (willrlin) [inline sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationAddress.tsx:17]: This is a partners page - is this an unrelated change?

File: sites/public/src/ListingView.tsx [hunk 2]
```diff
@@ -139,6 +139,21 @@ export const ListingView = (props: ListingProps) => {
     } else return t("listings.reservedCommunityTitleDefault")
   }
 
+  const shouldShowFeaturesDetail = () => {
```
Comment (willrlin) [inline sites/public/src/ListingView.tsx:142]: Does this take into account the additional fees section?

File: sites/partners/src/listings/PaperListingDetails/sections/DetailListingPhoto.tsx [hunk 3]
```diff
@@ -18,6 +18,8 @@ const DetailListingPhoto = () => {
   if (listing.image == null && listing.assets.length > 0) {
     listingFormPhoto = listing.assets.find((asset) => asset.label == "building")
   }
+  if (!listingFormPhoto) return <></>
```
Comment (willrlin) [inline sites/partners/src/listings/PaperListingDetails/sections/DetailListingPhoto.tsx:21]: Same question here

File: sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationAddress.tsx (hunk 1 above)
Comment (abbiefarr) [reply sites/partners/src/listings/PaperListingDetails/sections/DetailApplicationAddress.tsx:17]: Yeah, sorry this is unrelated, it came up when I was testing and was throwing errors so I just fixed it. Same below.

File: sites/partners/src/listings/PaperListingDetails/sections/DetailListingPhoto.tsx (hunk 3 above)
Comment (abbiefarr) [reply sites/partners/src/listings/PaperListingDetails/sections/DetailListingPhoto.tsx:21]: Ditto above

File: sites/public/src/ListingView.tsx (hunk 2 above)
Comment (abbiefarr) [reply sites/public/src/ListingView.tsx:142]: Good catch, added that in.

**Review by willrlin — APPROVED** [review]

**Pull request merged**